test-files/cmd/glacial/RndCropRandomCmdTest/out-tiles.osm

```diff
@@ -71,158 +71,142 @@
         <nd ref="-170915"/>
         <nd ref="-170917"/>
         <nd ref="-170914"/>
-        <tag k="error:circular" v="15"/>
+        <tag k="task" v="16"/>
         <tag k="node_count" v="639"/>
-        <tag k="boundary" v="task_grid_cell"/>
```

Comment:
Tiles output doesn't have to use `boundary=task_grid_cell` anymore to output the task grid, even in the OSM version.